CODEOWNERS

```diff
@@ -1,13 +1,14 @@
 *   @kevinbarabash @jeresig @somewhatabstract @jandrade @github-actions[bot]
 packages/wonder-blocks-breadcrumbs/*   @jandrade @kevinbarabash @jeresig
 packages/wonder-blocks-button/*   @kevinbarabash @jeresig
-packages/wonder-blocks-color/*   @jangmi-jo @kevinbarabash @jeresig
+packages/wonder-blocks-color/*   @kevinbarabash @jeresig
 packages/wonder-blocks-core/*   @somewhatabstract @kevinbarabash @jeresig
-packages/wonder-blocks-data/*   @somewhatabstract @jeresig @jangmi-jo
+packages/wonder-blocks-data/*   @somewhatabstract @jeresig
 packages/wonder-blocks-icon-button/*   @kevinbarabash @jeresig
 packages/wonder-blocks-link/*   @kevinbarabash @jeresig
 packages/wonder-blocks-modal/*   @jandrade @kevinbarabash @jeresig
 packages/wonder-blocks-popover/*   @jandrade @jeresig
+packages/wonder-blocks-testing/*   @somewhatabstract @jandrade @jeresig
```

Comment:
oh now that I see this I think we can get rid of CODEOWNERS now that we support Gerald in the repo. No need to change this, but more like we have to work on removing this from the repo.